polkadot-parachains/parachains-common/src/currency.rs

```diff
@@ -0,0 +1,13 @@
+//! Currency model used in encointer's runtimes.
```

Comment:
I think it is better to use chain-agnostic names, as we can reuse them accross all relay-chains.

However, we need to feature-gate `UNITS` when we start to support polkadot, as the atomic unit on polkadot has been changed to: `UNIT = 10_000_000_000`.

@brenzi better suggestion?

Reply:
sure. units is fine too